support/env/env.go

```diff
@@ -114,3 +115,21 @@ func CurrentAbsolutePath() string {
 
 	return res
 }
+
+func ModuleName() string {
```

Comment:
How about moving this function to packages/setup.go? There’s already a similar GetModulePath function there.